6_drivers_prep/code/nldas_feather_utils.R

```diff
@@ -0,0 +1,84 @@
+create_feather_task_makefile <- function(makefile, feather_files, include, feather_dir, ind_dir, packages, sources){
```

Comment:
is this function getting used somewhere?

Reply:
Not yet, it is partial, as I am starting to work on the feather/cell data effort